pom.xml

```diff
@@ -20,17 +20,13 @@
     <url>https://github.com/jenkinsci/mabl-integration-plugin</url>
 
     <properties>
-        <!-- Last Jenkins LTS release -->
-        <jenkins.version>1.625.1</jenkins.version>
-        <jenkins-test-harness.version>1.625.1</jenkins-test-harness.version>
+        <jenkins.version>2.222.4</jenkins.version>
+        <java.level>8</java.level>
+        <no-test-jar>false</no-test-jar>
+
         <release.skipTests>false</release.skipTests>
         <maven.javadoc.skip>true</maven.javadoc.skip>
-        <spotbugs-maven-plugin.version>4.0.0</spotbugs-maven-plugin.version>
         <concurrency>1</concurrency>
-        <java.level>7</java.level>
-        <hpi-plugin.version>2.7</hpi-plugin.version>
-        <workflow.version>1.14.2</workflow.version>
-        <hpi.compatibleSinceVersion>0.0.20</hpi.compatibleSinceVersion>
```

Comment:
@bertold Looks like I might have dropped this one in my pom example. I think we need to keep it. Right?

Reply:
@gone-skiing - yes. We have not released 0.0.20, so this is OK as is.